examples/own_dataset/README.md

```diff
@@ -0,0 +1,13 @@
+# Example of using your own dataset
+## Usage
+```
+python train.py dataset.csv --label value1 value2
+```
+
+## How to use your own dataset
```

Comment:
It reminds me that this is the only preferred way to use one's own dataset. But using `CSVFileParser` is just one way: Chainer Chemistry has `SDFFileParser` and users may even deal with the dataset without the parsers, although we think it would be costly. So how about changing the section title to "Procedure" simply?